RFC/src/RFC-0201_TariScript.md

```diff
@@ -370,19 +404,37 @@ pub struct BlockHeader {
     pub total_script_offset: BlindingFactor,
 }
 ```
-One important distinction to make is that the coinbase utxo does not count towards the _script offset_. This is because the coinbase UTXO already has special rules accompanying it and it has no input. Thus we cannot generate a \\( \so \\) for a coinbase transaction.
-The coinbase can allow any script and \\( k_{O} \\) as long as the range proof is validly constructed for \\( \hat{C_i} \\) and it does not break any of the rules in [RFC 120](RFC-0120_Consensus.md).
+
+This notion of the script offset \\(\so\\) means that the no third party can remove any input or output from a 
+transaction or the block, as that will invalidate the script offset balance equation, either (7) or (9) depending on 
+whether the scope is a transaction or block. It is important to know that this also stops 
+[cut&#8209;through](#cut-through) so that we can verify all spent UTXO scripts. Because the private script key and private 
```

Comment:
A party that knows `k_o` and `k_s` from an output being spent and the input spending it could still perform cut through. Not sure if this makes a difference, but probably worth mentioning

Reply:
While this is true, only the sender knows `k_s` and `k_o` so only he can do this.